graph/is_graph_bipartite.cpp

```diff
@@ -0,0 +1,146 @@
+/**
+ *  @file is_graph_bipartite
+ *
+ *	@brief Algorithm to check whether a graph is [bipartite](https://en.wikipedia.org/wiki/Bipartite_graph)
+ *	
+ *	@details
+ *	A graph is a collection of nodes also called vertices and these vertices
+ *	are connected by edges.A bipartite graph is a graph whose vertices can be
+ * 	divided into two disjoint and independent sets U and V such that every edge 
+ *	connects a vertex in U to one in V. 	
+ *	
+ *	The given Algorithm will determine whether the given graph is bipartite or not
+ *	
+ *	
+ * 	Example - Here is a graph g1 with 5 vertices and is bipartite
+ *	
+ *		1   4
+ *	   / \ / \
+ *	  2   3   5
+ *	
+ *	Example - Here is a graph G2 with 3 vertices and is not bipartite
+ *	
+ *		1 --- 2
+ *		 \   /
+ *		   3
+ *	
+ *	@author [Akshat Vaya](https://github.com/AkVaya)
+ *		
+ */
+#include <iostream>
+#include <vector>
+#include <queue>
+
+const int nax = 5e5 + 1;
+/**
+ * Class for representing graph as an adjacency list.
+ */
```

Comment:
Let us add a namespace to ensure all `graph` functions are all together.
You can also add a namespace member: namespace inside namespace.

For example, see [`backtracking/n_queens.cpp`](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/backtracking/n_queens.cpp) file.
```suggestion
 */
/**
 * @namespace graph
 * @brief Graph algorithms
 */
namespace graph {
```